### ISSUE TITLE
gnome.gnome-software: Failed to download screenshot

### ISSUE THREAD

**Issue by lucasew** [NixOS/nixpkgs#330932]
### Describe the bug

When opening the GNOME Software and selecting any app that has a screenshot it fails to load the screenshot and in the logs it says that TLS support is not available

```
jul 29 17:32:46 riverwood .gnome-software[3202943]: Failed to download screenshot: Suporte TLS não disponível
```

### Steps To Reproduce
1. nix run nixpkgs#gnome.gnome-software
2. Open any app page
3. Realize that screenshots are broken and journalctl has some `Failed to download screenshot` yellow messages

### Expected behavior
Screenshots loading properly

### Screenshots
![image](https://github.com/user-attachments/assets/4e7176b2-a15f-45db-b88c-f8185d4f884a)

### Additional context
Can be replicated in latest master

### Notify maintainers
@NixOS/gnome 

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
output here
```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by jtojnar** [comment]
This should have been fixed by https://github.com/NixOS/nixpkgs/pull/324559

**Comment by lucasew** [comment]
> This should have been fixed by #324559

It seems it is, but when I tried opening from master it seems that it keeps running in the background so it hit the previous session. Killing with kill and running again fixed the issue.
